### PR TITLE
fixup install.sh path insertion, Adds fixup section to crew to run during crew update.

### PR DESCRIPTION
**Pull request by satmandu**
- Fixes typo in install.sh
- Added a fixup section to crew which loads on crew update.

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=installer_fix CREW_TESTING=1 crew update
```

### CODE REVIEW

**Review by uberhacker — APPROVED** [review]
Nice find.

**Review by uberhacker — COMMENTED** [review]
Do we need to add something to update the broken `/usr/local/etc/env.d/path` for existing installs?

File: install.sh [hunk 1]
```diff
@@ -298,7 +298,7 @@ echo "LD_LIBRARY_PATH=$CREW_PREFIX/lib:$CREW_PREFIX/lib$LIB_SUFFIX" >> "$CREW_PR
 export LD_LIBRARY_PATH=$CREW_PREFIX/lib:$CREW_PREFIX/lib$LIB_SUFFIX
 
 # Add the CREW_PREFIX bin and musl bin directories to PATH.
-echo "PATH=$CREW_PREFIX/bin:$CREW_PREFIX/sbin:$CREW_PREFIX/share/musl/bin:$PATH" >> "$CREW_PREFIX"/etc/env.d/path
+echo -e "## Inserted by Chromebrew's install.sh\nPATH=$CREW_PREFIX/bin:$CREW_PREFIX/sbin:$CREW_PREFIX/share/musl/bin:\$PATH" > "$CREW_PREFIX"/etc/env.d/path
 export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/share/musl/bin:$PATH
```
Comment (uberhacker) [inline install.sh:302]: Don't we also need `$CREW_PREFIX/sbin` added?

Comment (uberhacker) [reply install.sh:302]: Oh, you snuck it in already.

File: lib/const.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Defines common constants used in different parts of crew
-CREW_VERSION = '1.36.3'
+CREW_VERSION = '1.36.5'
```
Comment (uberhacker) [inline lib/const.rb:2]: Only need to bump to `.4`?

Comment (satmandu) [reply lib/const.rb:2]: Intentional to fixup people testing...

Comment (satmandu) [reply lib/const.rb:2]: see new changes.

Comment (uberhacker) [reply lib/const.rb:2]: Who tests other than me or you?  Are we overthinking this? :)

File: lib/fixup.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Add fixups to be run during crew update here.
+
+# remove deprecated directory
+FileUtils.rm_rf "#{HOME}/.cache/crewcache/manifest"
+
+# Fix missing $PATH not added in install.sh
+if !system("grep -q '$PATH' #{CREW_PREFIX}/etc/env.d/path") || system("grep -q '1.36.4' #{CREW_PREFIX}/etc/env.d/path")
```
Comment (uberhacker) [inline lib/fixup.rb:7]: Not sure I follow the logic here.  We haven't merged a commit with `1.36.4` yet?

Comment (satmandu) [reply lib/fixup.rb:7]: It was only there during testing... it would have fixed your issue...

Comment (uberhacker) [reply lib/fixup.rb:7]: We don't actually have to merge this, do we?

Comment (satmandu) [reply lib/fixup.rb:7]: I can take it out... but it is nice to have the code in there as an example of fixing up an issue targeted to a version of a file...

Comment (uberhacker) [reply lib/fixup.rb:7]: If we leave it in, I will cringe every time I examine the code here in the future. :woozy_face:

**Review by uberhacker — APPROVED** [review]

**Pull request merged**